src/main/java/org/terasology/module/inventory/components/InventoryItemComponent.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.module.inventory.components;
+import com.google.common.collect.Lists;
+import org.terasology.engine.entitySystem.prefab.Prefab;
+import org.terasology.engine.world.block.Block;
+
+import java.util.List;
+
+/**
+ * A simple class connecting a resource (a {@link Block} or {@link Prefab}) to a quantity
+ */
+public class InventoryItemComponent {
```

Comment:
This is not a component in the ECS sense, but only a data class. In particular, instances of this class should not be used as components.

This is also important as components should not refer directly to other components.

```suggestion
public class InventoryItem {
```

Reply:
Ok will make the change I just wasn't too sure how a class would fit into the ECS architecture.